onnx/checker.py

```diff
@@ -24,8 +24,13 @@
 from typing import TypeVar, Callable, Any, Type, cast, Union, Text
 from six import string_types
 import onnx.shape_inference
+import sys
+import warnings
 
 
+# Limitation of single protobuf file is 2GB
+MAXIMUM_PROTOBUF = 1900000000  # loose boundary
```

Comment:
Why do we need a "loose" boundary? Can we define it exactly?

Reply:
I was afraid that the function is approximate. Just did some experiment show the calculation is precise enough. Should set 2GB for the readability. Thank you for catching this.